Resources/Audio/Effects/Footsteps/attributions.yml

```diff
@@ -22,6 +22,7 @@
 
 - files:
   - slime1.ogg
+  - slime2.ogg # the sound is quieter
```

Comment:
The footsteps no longer override the sound params with a recent fix, so now you can tweak the sound in the prototype instead of repeating the sound file, i.e
```
        insertSound:
          path: /Audio/Effects/Vehicle/vehiclestartup.ogg
          params:
            volume: -3
```

Reply:
fixed